<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadAutoSaveImageDialog</class>
 <widget class="QDialog" name="LoadAutoSaveImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load AutoSaveImage</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="icon">
    <iconset resource="AutoSaveImageListForSearch.qrc">
     <normaloff>:/Qtres/Load.png</normaloff>:/Qtres/Load.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>570</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="AutoSaveImageListForSearch.qrc">
     <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>541</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Drive</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAllDrives">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>All drives</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSelectDrive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>D Drive (D:\)</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidgetFound">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>311</width>
      <height>351</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Date/Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MasterID</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QDateTimeEdit" name="dateTimeEditFrom">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEditTo">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>From</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>To</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonStartSearch">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start search</string>
    </property>
    <property name="icon">
     <iconset resource="AutoSaveImageListForSearch.qrc">
      <normaloff>:/Qtres/Search.png</normaloff>:/Qtres/Search.png</iconset>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="AutoSaveImageListForSearch.qrc"/>
 </resources>
 <connections/>
</ui>
